mkdocs/docs/HPC/setting_up_python_virtual_environments.md

```diff
@@ -0,0 +1,221 @@
+# Python Virtual Environments (venv's)
+
+## Introduction
+
+Python virtual environments (venv's) are a way to create isolated Python environments.
+You can install packages in a venv without affecting the system Python installation.
+Because a normal user cannot install packages globally, 
+venv's are a way to install packages without needing root access.
```

Comment:
`root access` -> `administrator privileges`?

Reply:
Fixed in [87a30af](https://github.com/hpcugent/vsc_user_docs/pull/639/commits/87a30afc16070d39726aa37139fbdc62b1162432)